lighthouse-core/audits/script-treemap-data.js

```diff
@@ -213,7 +213,15 @@ class ScriptTreemapDataAudit extends Audit {
             unusedBytes: unusedJavascriptSummary.sourcesWastedBytes[source],
           };
 
-          const key = ModuleDuplication.normalizeSource(source);
+          // ModuleDuplication uses keys without the source root prepended, but
```

Comment:
Noticed an issue with the duplication stuff. Need to make a separate PR for this.

Reply:
First one. This is the fix.